src/plugins/embeddable/public/lib/embeddables/common/constants.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const COMMON_EMBEDDABLE_GROUPING = {
```

Comment:
nit: can we type these? Maybe `UiActionsPresentableGrouping<unknown>`?

Reply:
Updated the type in [5337820](https://github.com/elastic/kibana/pull/183764/commits/533782083212a546263114859366babff6af3bd7)